config/webhook/manifests.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: mutating-webhook-configuration
+webhooks:
+- admissionReviewVersions:
+  - v1
+  clientConfig:
+    service:
+      name: webhook-service
+      namespace: remotesecret
+      path: /mutate-appstudio-redhat-com-v1beta1-remotesecret
+    caBundle: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSURHVENDQWdHZ0F3SUJBZ0lVUWlNaUthaG12b0M1OThFVVZHRHhDQkgzTk93d0RRWUpLb1pJaHZjTkFRRUwKQlFBd0d6RVpNQmNHQTFVRUF3d1FjbVZ0YjNSbGMyVmpjbVYwTG5OMll6QWdGdzB5TXpBM01UY3dPREUwTXpkYQpHQTh5TWprM01EVXdNVEE0TVRRek4xb3dHekVaTUJjR0ExVUVBd3dRY21WdGIzUmxjMlZqY21WMExuTjJZekNDCkFTSXdEUVlKS29aSWh2Y05BUUVCQlFBRGdnRVBBRENDQVFvQ2dnRUJBS01zdm9LNU1FZ3Q4THNwWkVicVBzS1YKcUtoOGJZVElFb1NiK0dSWEdXQ3gweDArL0IvUlJFZ2IyWXQ3aytuVnY5K2JZMmo5eHRwN0ljRUJDYmRCd1oyWgpiSEpoQmJiWEFSYXBKL0QzZXJHSXcxM2hJOEFZWmxWOWpPUlphVGJJSk16UUlwSnl2c0EzNXlQVnRNSUgxeHNJCm9iRFByTWlJRm1EUnY0d2Q2S1FJUVllQkM0TnZpRzROcElveTRWZnNPdjlvOXY1K1pNM0hBYWRjSmNFSjY2bDgKaWlwd0lCRitZT2kyN0NUdW9Iajl0R2hFTHJxZ2UyVklUd1lweC9XNXhNdFg4R0djNCt6b1FEQkRiV2NKSDFjYQppWXNta2xkSzRoYi9sUEpJMEpEeDBIU1JXV1dPeVZiZkovZE9DUjgyYkI5MWNWelJaN3N3cHNPNnlLZkZ4ZWNDCkF3RUFBYU5UTUZFd0hRWURWUjBPQkJZRUZDNmpvUDZJZDVjTUYzK0paeXlvVmRIQjFiRHJNQjhHQTFVZEl3UVkKTUJhQUZDNmpvUDZJZDVjTUYzK0paeXlvVmRIQjFiRHJNQThHQTFVZEV3RUIvd1FGTUFNQkFmOHdEUVlKS29aSQpodmNOQVFFTEJRQURnZ0VCQURpRWhjOFpPaktaZS96Y1FyYkJiZmlGRnpyL0UrQUJRcXlvb2NTbXo1NG01YVlRCmxnSC9NdnozK2k4SEkzVVlBa3I5T1pvLzE0S0RLZ3BOSnA4eEs2TUZjZmFQZlhzWGJzanFJcnIwNjJHY1hkckUKcGlpeVJ0L20zdVgzL0VWV0ZnSStJSUVsUU5TZkJ0YjE1WkQxWGhqaTBPdCsrcGV5ZEdMNHVteGVsSDMvMmwxVApPMVZXVU1zd1lDcC9xdjdmUUFucmxJbUJFVUtVVlk3cDVzQ0VrM2M4R0pMOGxUTzJJSTdCck1QSmc4bVNPTGQyCmkvMy9mSmVvbk1yeCtZdTBmbDlkbmRPTnlrZUFLNkJ4akMvdE9mSUxZaW9BNFpMQVVLRjIyd0tDR09iK28vT1cKbkpGSVFzZ3ZoOGh1UGF2VkR5MllDWWQydm0xek5pSzZxZ0tmRWdJPQotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tCg==
```

Comment:
This needs to be either generated or referenced (e.g. referencing the autogenerated certs on OpenShift and reference certmanager's certificate on plain Kubernetes).

Reply:
Agree (for all 3), let's script creating of this, the thing I'd like to avoid is installing cert-manager.